website/client/components/chat/reportFlagModal.vue

```diff
@@ -106,14 +106,16 @@ export default {
       this.$root.$emit('bv::hide::modal', 'report-flag');
     },
     async reportAbuse () {
-      this.notify('Thank you for reporting this violation. The moderators have been notified.');
+      this.text('Thank you for reporting this violation. The moderators have been notified.');
```

Comment:
For group messages, that text is good. For private messages, can it be this:
"Thank you for reporting this message. The moderators have been notified. You can now delete the message if you wish."

Also can that be a translatable string?